### PR TITLE
Update version to v1.0.0-beta.

### PR DESCRIPTION
**Pull request by tmeyer2115**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (nmanu1) [inline package.json:1]: can we point to the new Headless-React version?

**Review by nmanu1 — COMMENTED** [review]
does the PR description need release notes?

File: package.json (hunk 1 above)
Comment (tmeyer2115) [reply package.json:1]: Updated!

**Comment by tmeyer2115** [comment]
> does the PR description need release notes?

I'm going to add a brief note when I make the official GH Release. It's basically going to just point to the docs. I don't think the PR/Commit needs a message.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
   "name": "@yext/search-ui-react",
-  "version": "0.2.0-beta.243",
+  "version": "1.0.0-beta",
```
Comment (oshi97) [inline package.json:3]: I think I still prefer it if we have the PR number here.

Comment (tmeyer2115) [reply package.json:3]: Product asked for this to be the version. I can see if they're okay with us adding the PR number.

**Review by oshi97 — APPROVED** [review]

**Pull request merged**